.github/workflows/ci.yaml

```diff
@@ -109,7 +136,7 @@ jobs:
                   chmod 0600 engflow.crt engflow.key
                   echo "$ENGFLOW_CLIENT_CRT" > engflow.crt
                   echo "$ENGFLOW_PRIVATE_KEY" > engflow.key
-                  echo "USE_BAZEL_VERSION=6.0.0-pre.20220922.1" >> $GITHUB_ENV
+                  echo "USE_BAZEL_VERSION=${{ matrix.bazelversion }}" >> $GITHUB_ENV
```

Comment:
could we just remove this line now that it's already set in the GITHUB_ENV above?

Reply:
👍